src/reanimated2/animation/spring.ts

```diff
@@ -56,6 +78,15 @@ export function withSpring(
         return true;
       }
 
+      if (config.configIsInvalid) {
+        // We don't animate wrong config
+        if (config.useDuration) return false;
```

Comment:
I am afraid if this condition could be a cause of infinity loop. I think that we should end the animation here, why you want to continue animation?